test/MUIDataTableToolbar.test.js

```diff
@@ -241,4 +240,30 @@ describe('<TableToolbar />', function() {
 
     assert.strictEqual(onDownload.callCount, 1);
   });
+
+  it('should trigger onDownload prop callback when calling method handleCSVDownload and skip download if return from onDownload is false', () => {
```

Comment:
This test doesn't prove that the download is skipped, so I'm not sure it's doing anything for us that the other test for `onDownload` isn't already doing. I think that the way the CSV creation works at the moment, it's difficult to determine in a test whether or not the download was successful (just whether or not the callback fired). To do so, we'd probably need to expose another callback that fires after successful download and returns the data, but this is outside the scope of this PR, and I'm not sure we need it at this point.

Long story short, I appreciate the attempt to add to our coverage, but I think it would better to remove this test for now.

Instead, if you want, we could add to the `csv-export` example a conditional that would trigger this option and prevent download, so at least we'd have a built in way to verify in the codebase. If you don't want to do that, that's fine, I can add it after this gets merged in, before the next release.

Reply:
Removed the test and added `onDownload: () => false` to the example!